### DISCUSSION TITLE
Duplicates in docs

### DISCUSSION THREAD

**Discussion by hedgehog319** [LazyVim/LazyVim#2284] [Q&A] [answered]
In the [**plugins/ui**](https://www.lazyvim.org/plugins/ui) section **dressing.nvim** repeated three times, is that correct?
![image](https://github.com/LazyVim/LazyVim/assets/38129787/01bd9ce0-4d2d-4445-be79-7a3cb48c686c)

**Comment by dpetka2001** [comment, marked as answer]
That looks like a glitch to me. As you can see in the [code](https://github.com/LazyVim/LazyVim/blob/879e29504d43e9f178d967ecc34d482f902e5a91/lua/lazyvim/plugins/ui.lua#L44-L52) it's being referenced inside its own spec, but my guess is that it gets mistakenly being picked up as 2 additional fragments of spec by the auto-generation build system.

**Reply by hedgehog319** [reply]
Understood, thanks for your answer)
